README.md

```diff
@@ -59,6 +59,10 @@ Saved 272 results into data/output.example.csv
 If you have any questions, contact @littlepea
 
 ## Running tests
+You will need to install pytest 
+```shell
+pip install pytest
+```
```

Comment:
I think there's no need to `pip install`, pytest is already a poetry dependency...
Instead, maybe we should change the command below to `poetry run pytest`

Reply:
updated